src/solver/td_simplified.ml

```diff
@@ -0,0 +1,233 @@
+(** Terminating op-down solver with side effects. Baseline for comparisons with td_parallel solvers ([td_simplified]).*)
+
+(** Top down solver that uses the box-operator for widening/narrowing at widening points.
+ * Options:
+ * - solvers.td3.remove-wpoint (default: true): Remove widening points when a variable is and stays stable in iterate. Increases precision of nested loops.*)
```

Comment:
I'm not sure how much we want to iterate on this. This solver isn't too important to us, but it's supposed to be didactic, so maybe we should spend time polishing this.
The following thoughts are first for discussion than immediate requests for changes.

Anyway, if it's "terminating" then it cannot be the warrowing version (which is non-terminating). The first improvement of TD3 is the termination by switching from warrowing to widen/narrow phases. Personally, I would prefer the phased version instead of warrowing: it's not really more complicated (just one argument to `solve`).

As to the `remove-wpoint` improvement, I wouldn't have it in this simplified version because it is niche.